INFO - 2023-03-19 07:58:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-03-19 07:58:45 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'hammah54_z'@'localhost' (using password: YES) in C:\xampp\htdocs\bc\myapp\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\bc\myapp\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'hammah54_z', 'H70724742@m', 'hammah54_z', 3306, '', 0)
#1 C:\xampp\htdocs\bc\myapp\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\bc\myapp\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\bc\myapp\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\bc\myapp\app\Controllers\HomeController.php(13): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\bc\myapp\system\CodeIgniter.php(934): App\Controllers\HomeController->index()
#6 C:\xampp\htdocs\bc\myapp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#7 C:\xampp\htdocs\bc\myapp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\bc\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\bc\myapp\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\bc\myapp\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\bc\myapp\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\bc\myapp\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\bc\myapp\app\Controllers\HomeController.php(13): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\bc\myapp\system\CodeIgniter.php(934): App\Controllers\HomeController->index()
#5 C:\xampp\htdocs\bc\myapp\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#6 C:\xampp\htdocs\bc\myapp\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\bc\index.php(67): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-03-19 07:58:45 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `home`', [], false)
 3 APPPATH\Controllers\HomeController.php(13): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\HomeController->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
